addons/resource.language.en_gb/resources/strings.po

```diff
@@ -4867,21 +4867,57 @@ msgctxt "#10008"
 msgid "Play next"
 msgstr ""
 
-#empty strings from id 10009 to 10010
+#. Header label about yes/no dialog to confirm remove of part
+#: xbmc/addons/gui/GUIDialogAddonSettings.cpp
+msgctxt "#10009"
+msgid "Confirm add-on configuration remove"
+msgstr ""
+
+#. Description label about yes/no dialog to confirm remove of part, header defined by #10009
+#: xbmc/addons/gui/GUIDialogAddonSettings.cpp
+msgctxt "#10010"
+msgid "Are you sure to remove the selected add-on configuration #{0:d}?"
+msgstr ""
 
 #: xbmc/guilib/WindowIDs.h
 msgctxt "#10011"
 msgid "Settings - Videos - Screen calibration"
 msgstr ""
 
-#empty strings from id 10012 to 10015
+#. Header label about selection dialog within addon info to select specific instance settings
+#: xbmc/addons/gui/GUIDialogAddonSettings.cpp
+msgctxt "#10012"
+msgid "Add-on configuration selection"
+msgstr ""
+
+#. Button inside selection dialog to get basic addon settings (without instance part), #10012 is defined dialog header
+#: xbmc/addons/gui/GUIDialogAddonSettings.cpp
+msgctxt "#10013"
+msgid "Base settings"
+msgstr ""
+
+#. Button inside selection dialog to add an new instance, #10012 is defined dialog header
+#: xbmc/addons/gui/GUIDialogAddonSettings.cpp
+msgctxt "#10014"
+msgid "Add add-on configuration"
+msgstr ""
+
+#. Button inside selection dialog to remove an present instance, #10012 is defined dialog header
+#: xbmc/addons/gui/GUIDialogAddonSettings.cpp
+msgctxt "#10015"
+msgid "Delete add-on configuration"
```

Comment:
"Remove add-on configuration" (matches "add" used elsewhere and the wording used in the removal confirmation dialog)